lambdas/service/app.py

```diff
@@ -204,23 +206,12 @@ def swagger_ui():
     'tags': ['Auxiliary']
 })
 def openapi():
-    gateway_id = app.current_request.context['apiId']
-    raw_spec = aws.api_gateway_export(gateway_id)
-    spec = app.annotated_specs(raw_spec, openapi_spec)
+    spec = app.annotated_specs(openapi_spec)
     return Response(status_code=200,
                     headers={"content-type": "application/json"},
                     body=spec)
 
 
-@app.route('/openapi/raw')
```

Comment:
Since an API route is being removed, maybe the API version should have a minor bump? This a question, not a requested change. I don't know the protocol for this, and I acknowledge that this change is reasonably not going to affect any azul clients.

Reply:
I don't know the answer to this either. @hannes-ucsc can you weigh in? Do we even care about maintaining this API version identifier?

Reply:
Generally, speaking yes. And removal of a route would justify a major version update. But since this is an internal endpoint and since we're still in the phase of making our API publicly available, I wouldn't bother this time around.